pubspec.yaml

```
@@ -4,13 +4,13 @@ description: A pluggable, mockable process invocation abstraction for Dart.
 homepage: https://github.com/google/process.dart
 
 environment:
-  sdk: '>=2.14.0-0 <3.0.0'
+  sdk: '>=2.14.0-0 <4.0.0'
 
 dependencies:
-  file: '^6.0.0'
+  file: '^7.0.0'
   path: ^1.8.0
   platform: '^3.0.0'
 
 dev_dependencies:
-  lints: ^1.0.1
+  lints: ^2.1.0
```

```suggestion
  lints: '>=1.0.1 <3.0.0'
```
Same for lints: `lints: '>=1.0.1 <3.0.0'`